src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -368,15 +379,25 @@ enum class DlCredentialStatus : uint8_t
     kOccupied  = 0x01, /**< Indicates if credential slot is already occupied. */
 };
 
+enum class DlAssetSource : uint8_t
+{
+    kUnspecified = 0x00,
+    kMatterIM    = 0x01,
+};
+
 /**
  * @brief Structure that holds the credential information.
  */
 struct EmberAfPluginDoorLockCredentialInfo
 {
-    DlCredentialStatus status;        /**< Indicates if credential slot is occupied or not. */
-    DlCredentialType credentialType;  /**< Specifies the type of the credential (PIN, RFID, etc.). */
-    chip::ByteSpan credentialData;    /**< Credential data bytes. */
-    chip::FabricIndex createdBy;      /**< ID of the fabric that created the user. */
+    DlCredentialStatus status;       /**< Indicates if credential slot is occupied or not. */
+    DlCredentialType credentialType; /**< Specifies the type of the credential (PIN, RFID, etc.). */
+    chip::ByteSpan credentialData;   /**< Credential data bytes. */
+
+    DlAssetSource creationSource;
+    chip::FabricIndex createdBy; /**< ID of the fabric that created the user. */
```

Comment:
```suggestion
    chip::FabricIndex createdBy; /**< Index of the fabric that created the user. */
```
and similar for the other members.